File: Programming Basics with Python/02 Conditional statements/More exercises/03_Harvest.py
import math

X = int(input())
Y = float(input())
Z = int(input())
workers = int(input())
for_wine = X * 0.4
grapes_for_wine = for_wine * Y
liters_wine = grapes_for_wine / 2.5
result = liters_wine - Z
if Z > liters_wine:
    print(f'It will be a tough winter! More {math.floor(abs(result))} liters wine needed.')
else:
    print(f'Good harvest this year! Total wine: {math.floor(liters_wine)} liters.')
    print(f'{math.ceil(result)} liters left -> {math.ceil(result / workers)} liters per person.')

File: JS Basics/02 Conditional Statements - More Exercises/03. Harvest.js
function vineyard(area, grapesPerMeter, wineNeeded, workers) {
    let forWine = Number(area) * 0.4;
    let wine = forWine * Number(grapesPerMeter) / 2.5;
    let result = Math.abs(wine - wineNeeded)
    if (wine < Number(wineNeeded)) {
        console.log(`It will be a tough winter! More ${Math.floor(result)} liters wine needed.`);
    }
    else {
        console.log(`Good harvest this year! Total wine: ${Math.floor(wine)} liters.`);
        console.log(`${Math.ceil(result)} liters left -> ${Math.ceil(result / Number(workers))} liters per person.`);
    }

}
vineyard(650, 2, 175, 3)